.expeditor/scripts/bump_version.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# This script automates the modification of our Homebrew tap for the
+# `hab` binary.
+
+set -euo pipefail
+
+if ! sed --version 2>&1 | grep -q "GNU sed"; then
+    echo "This script requires GNU sed; aborting"
+    exit 1
+fi
```

Comment:
We could probably just dispense with this. If GNU `sed` isn't in the Expeditor containers this is going to run in, we should get it in there.